iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/AbstractTreeDataExporter.java

```diff
@@ -185,6 +194,42 @@ private void addExtensionsVersions(Properties parameters, ExportOptions options)
         });
     }
 
+    private void addTopologyLevelVoltageLevels(Properties parameters, ExportOptions options) {
+        List<String> nodeBreakerVoltageLevelsList = Parameter.readStringList(getFormat(), parameters, VOLTAGE_LEVELS_NODEBREAKER_PARAMETER, defaultValueConfig);
+        List<String> busBreakerVoltageLevelsList = Parameter.readStringList(getFormat(), parameters, VOLTAGE_LEVELS_BUSBREAKER_PARAMETER, defaultValueConfig);
+        List<String> busBranchVoltageLevelsList = Parameter.readStringList(getFormat(), parameters, VOLTAGE_LEVELS_BUSBRANCH_PARAMETER, defaultValueConfig);
+
+        List<String> allVoltageLevelIds = new ArrayList<>();
+        allVoltageLevelIds.addAll(nodeBreakerVoltageLevelsList);
+        allVoltageLevelIds.addAll(busBreakerVoltageLevelsList);
+        allVoltageLevelIds.addAll(busBranchVoltageLevelsList);
+
+        for (String voltageLevelId : allVoltageLevelIds) {
+            int foundVoltageLevelId = 0;
+            TopologyLevel topologyLevel = null;
+
+            if (nodeBreakerVoltageLevelsList.contains(voltageLevelId)) {
+                foundVoltageLevelId++;
+                topologyLevel = TopologyLevel.NODE_BREAKER;
+            }
+            if (busBreakerVoltageLevelsList.contains(voltageLevelId)) {
+                foundVoltageLevelId++;
+                topologyLevel = TopologyLevel.BUS_BREAKER;
+            }
+            if (busBranchVoltageLevelsList.contains(voltageLevelId)) {
+                foundVoltageLevelId++;
+                topologyLevel = TopologyLevel.BUS_BRANCH;
+            }
+
+            if (foundVoltageLevelId == 1) {
+                options.addVoltageLevelTopologyLevel(voltageLevelId, topologyLevel);
+            } else {
+                String warnNotUniqueVoltageLevel = String.format("VoltageLevel %s is associated with different topology levels in property : we ignore it", voltageLevelId);
+                LOGGER.warn(warnNotUniqueVoltageLevel);
```

Comment:
This can be written in one line:

```suggestion
                LOGGER.warn("VoltageLevel {} is associated with different topology levels in property => ignored", voltageLevelId);
```

Note also that I slightly changed the message.